test/queryFilters/interpretExpression.test.ts

```diff
@@ -10,7 +10,7 @@ const complexQueryELM = getELMFixture('elm/queries/ComplexQueries.json');
 const PATIENT = FHIRWrapper.FHIRv401().wrap({
   resourceType: 'Patient',
   birthDate: '1988-09-08'
-}) as CQLPatient;
+}) as any as CQLPatient;
```

Comment:
It looks like the `wrap()` function used to return type `any` but now returns type `FHIRObject`, which is why we need to cast to `any`. @mgramigna do you think this is an okay approach so that we can still cast to type `CQLPatient`? (asking you since you made the return type change to `wrap()`)

Reply:
Haha yeah I ok-ed this because of how CQLPatient is defined. I think in the future there's a chance we can unify it better with the cql-exec-fhir types, but I don't think we should worry about it now.

The `as any` was to avoid tsserver yelling about insufficient overlap during a type cast. Happens to the best of us